mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -450,6 +450,32 @@ func TestAccBigQueryDataTable_jsonEquivalency(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryDataTable_canReorderParameters(t *testing.T) {
+	t.Parallel()
+
+	datasetID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	tableID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				// we don't run any checks because the resource will error out if
+				// it attempts to destory/tear down.
+				Config: testAccBigQueryTable_jsonPreventDestroy(datasetID, tableID),
+			},
+			{
+				Config: testAccBigQueryTable_jsonPreventDestroyOrderChanged(datasetID, tableID),
+			},
+			{
+				Config: testAccBigQueryTable_jsonEq(datasetID, tableID),
```

Comment:
Do we need this step? It's my understanding that we don't because the implicit delete step runs with no config and prevent_destroy is not tracked in state. However the test environment is a little different than real Terraform.

Reply:
confirmed that we do need this step